paths/reset/07-git-reflog.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: simple-class
+header:
+  overlay_image: cover.jpeg
+  overlay_filter: rgba(46, 129, 200, 0.6)
+title: Reflog and Cherry Pick
+permalink: /reset/07
+next-page: /reset/08
+facilitator: false
+sidebar:
+  nav: "reset"
+main-content: |
+  So, we just had you delete all of the files you created and committed! However, we can bring them back using two commands, `git reflog` and `git cherry-pick`.
+
+  1. Enter `git reflog` and identify the SHA-1 with the commit message `Add files 02 03 and 04`.
+  1. Enter `git cherry-pick <SHA-1>`, where <SHA-1> is the SHA-1 associated with the commit `Add files 02 03 04`.
+
+show-me-how:
+tell-me-why: |
+  ## Reflog
+  Similar to `git log`, `git reflog` shows the local history of your repository by identifying when branches and other references were updated and can be used to alter the history of your project. Keep in mind, the contents of your `reflog` isn't pushed to the remote, so, it can't be used by other collaborators.
```

Comment:
Reflog is a history of every commit where HEAD has pointed. Small difference in syntax that could add clarity.